packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss

```diff
@@ -1,28 +1,19 @@
 @import '~carbon-components/scss/globals/scss/layout';
 
 .page-header {
-  background: $carbon--black-100;
+  background: $ui-05;
   color: $inverse-01;
   width: 100%;
   display: flex;
   flex-direction: column;
   justify-content: flex-end;
-  height: 20rem;
```

Comment:
is this height change on purpose?

Reply:
Yeah the tabs are removed

Reply:
It's supposed to be the same height with or without tavs, the 20rem is correct. Unless there is an updated design somewhere??

Reply:
Ah, unintentional. To change the height now, you'll just need to set the height custom property. It'll shorten it by 3rem if there's tabs.

```css
--height: 14rem
```